cmd/snap-repair/cmd_done_retry_skip_test.go

```diff
@@ -63,7 +64,11 @@ func (r *repairSuite) TestStatusHappy(c *C) {
 		defer rp.Close()
 		defer wp.Close()
```

Comment:
Do we need this defered wp.Close() still?

Reply:
it doesn't hurt, it's very similar to the code in Run itself effectively, if you think about it,
but given that is dupped we could survive trusting the finalizer in the worst case now